tests/trilinos_tpetra/08.cc

```diff
@@ -0,0 +1,98 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2004 - 2018 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
```

Comment:
This and the other tests still have the old headers. Would you please replace this by the current version https://github.com/dealii/dealii/blob/add8e16ae6718e1e0521653ec1f5c1e48db5ee57/tests/base/aligned_vector_01.cc#L1-L13 (just picking a random file here).

Reply:
Done!